frontend/src/components/limited-textarea/index.tsx

```diff
@@ -0,0 +1,36 @@
+import { useState } from "react";
+import { Text, Textarea, TextareaProps } from "theme-ui";
+
+type LimitedTextareaPros = TextareaProps & {
+  limit: number;
+};
+export const LimitedTextarea = ({
+  value,
+  limit,
+  onChange,
+  ...props
+}: LimitedTextareaPros) => {
+  const [content, setContent] = useState(value.slice(0, limit));
+
+  return (
+    <>
+      <Textarea
+        onChange={(event) => {
+          onChange(event);
+          setContent(event.target.value.slice(0, limit));
```

Comment:
I don't think this is correct

the local Content will be updated to have the "sliced" version, but we will still send the long value to the component upstream (the CFP Form) because we do `onChange(event)` which will take the content from `event`, so when sending the form to GraphQL you would still send the long value causing an error.